projects/laji/src/app/shared-modules/observation-result/selected-field-group/selected-field-group.component.ts

```diff
@@ -108,6 +93,16 @@ export class SelectedFieldGroupComponent {
     }
   }
 
+  getFieldColumnArrayLength(field: string) {
+    if (/gathering\.conversions\.(wgs84|euref|ykj)(CenterPoint)\./.test(field)) {
+      return 2;
+    } else if (/gathering\.conversions\.(wgs84|euref|ykj)\./.test(field)) {
+      return 4;
+    } else {
+      return 1;
+    }
+  }
```

Comment:
Works ok right now, but if more code is added that tries to check width for a `field` of value `undefined`, this will return `1` which could result in bugs. So a type check for the `field` is needed, and `0` should be the answer in that case.

Reply:
Actually, we don't need to hard code the array length, could you compute the values from  `this.getFieldColumnArray()`? Something like this:

```
getFieldColumnArrayLength(field: string) {
  return this.getFieldColumnArray(field)?.length || 0;
}
```

Reply:
I think your recommendation should work as-is, so I'll drop it in, and check that getFieldColumnArray has some sanity check for undefined field names.